src/wrapper.test.js

```diff
@@ -39,138 +74,113 @@ describe('Distro initialization', () => {
 
   afterEach(() => {
     process.env = ORIGINAL_PROCESS_ENV;
-    OTLPTraceExporter.mockClear();
+    jest.resetAllMocks();
   });
 
   describe("with the 'LUMIGO_SWITCH_OFF' environment variable set to 'true'", () => {
-    it('should not invoke trace initialization', async () => {
+    test('should not invoke trace initialization', async () => {
       process.env.LUMIGO_SWITCH_OFF = 'true';
 
-      jest.isolateModules(async () => {
+      await jest.isolateModulesAsync(async () => {
         const wrapper = jest.requireActual('./wrapper');
 
-        await wrapper.init.then((sdkInitialized) => {
-          expect(OTLPTraceExporter).not.toHaveBeenCalled();
-          expect(sdkInitialized).toBeUndefined();
-        });
+        const sdkInitialized = await wrapper.init;
+
+        expect(OTLPTraceExporter).not.toHaveBeenCalled();
+        expect(sdkInitialized).toBeUndefined();
       });
     });
   });
 
-  describe('with an initialization failure', () => {
+  describe('secret keys', () => {
     beforeEach(() => {
-      process.env.LUMIGO_ENDPOINT = LUMIGO_ENDPOINT;
-      process.env.LUMIGO_TRACER_TOKEN = LUMIGO_TRACER_TOKEN;
+      /*
+       * We have a limit on the size of env we sent to the backend, and the env
+       * in the CI/CD goes over the limit, so the additional env vars we want to
+       * check for scrubbing get dropped.
+       */
+      process.env = {};
     });
 
-    test('should reject the init promise', () => {
```

Comment:
why those tests were removed?

Reply:
Because they never worked. I found out by fixing the entire test harness.